scripts/monthly_master_script.sh

```diff
@@ -244,4 +244,24 @@ fi
 # Remove after checking for successful completion
 rm success.log
 
+### backing up the Zenodo backup
+# Log file saved here: ./logs/upload_zenodo_files.log
+# -------------------------------------
+
+echo "$(date -Is) : Uploading files for Zenodo..." >> $MASTER_LOG
+$PYTHON_ENV scripts/update_zenodo.py
+if [ -e success.log ]; then
+   echo "$(date -Is) : SUCCESS." >> $MASTER_LOG
+else
+   echo "$(date -Is) : FAILED. Exiting <${SCRIPT_NAME}>." >> $MASTER_LOG
+   exit 1
+fi
+# Remove after checking for successful completion
+rm success.log
+
+
 echo "$(date -Is) : Script complete." >> $MASTER_LOG
+
+git add data/iffy_files
```

Comment:
This is a nice add. However, can you please log the addition in the same way as the other commands? You can even write the git line output directly to the master log.

e.g. ...
```bash
echo "$(date -Is) : Updating..." >> $MASTER_LOG
git add data/iffy_files >> $MASTER_LOG
...
```

As far as I know, this should work.

Then you can move the `echo "$(date -Is) : Script complete." >> $MASTER_LOG` line after this block. Thanks!

Reply:
block added👍🏻 Thanks!